R/Shiny.R

```diff
@@ -110,128 +110,59 @@ launchDiagnosticsExplorer <- function(dataFolder = "data",
     connectionDetails = connectionDetails,
     resultsDatabaseSchema = resultsDatabaseSchema,
     vocabularyDatabaseSchemas = vocabularyDatabaseSchemas,
-    dataFolder = dataFolder,
-    dataFile = dataFile,
     aboutText = aboutText
   )
   .GlobalEnv$shinySettings <- shinySettings
   on.exit(rm("shinySettings", envir = .GlobalEnv))
   shiny::runApp(appDir = appDir)
 }
 
-#' Premerge Shiny diagnostics files
+#' Merge Shiny diagnostics files into sqlite database
 #'
 #' @description
-#' This function combines diagnostics results from one or more databases into a single file. The result is a
-#' single file that can be used as input for the Diagnostics Explorer Shiny app.
+#' This function combines diagnostics results from one or more databases into a single file. The result is an
+#' sqlite database that can be used as input for the Diagnostics Explorer Shiny app.
 #'
 #' It also checks whether the results conform to the results data model specifications.
 #'
 #' @param dataFolder       folder where the exported zip files for the diagnostics are stored. Use
 #'                         the \code{\link{runCohortDiagnostics}} function to generate these zip files.
 #'                         Zip files containing results from multiple databases may be placed in the same
 #'                         folder.
-#' @param tempFolder       A folder on the local file system where the zip files are extracted to. Will be cleaned
-#'                         up when the function is finished. Can be used to specify a temp folder on a drive that
-#'                         has sufficient space if the default system temp space is too limited.
-#'
+#' @param sqliteDbPath     Output path where sqlite database is placed
+#' @param overwrite        (Optional) overwrite existing sqlite lite db if it exists.
 #' @export
-preMergeDiagnosticsFiles <-
```

Comment:
i am not sure what the best practice is -- if someone has an old script with preMergeDiagnosticsFiles, would they just get a function not found type of error? or should we keep a preMergeDiagnosticsFiles function -- all it would do is give an informative stop error and recommend the use of createMergedResultsFile function